reference/robot.md

```diff
@@ -500,7 +501,7 @@ in the user guide).
 
 The integers can be compared to the following enumeration items:
 
-%figure "Helper enumeration to interpret the integer argument and return value of the *wb_robot_[gs]et_mode()* functions"
+%figure "Helper enumeration to interpret the integer argument and return value of the *wb_robot_[gs]et_mode* functions"
```

Comment:
Sometimes in figure caption the function is formatted using \*\* and sometimes with "\`<name>\`".
Is there a rule for this?